docsets/Agora_Objective-C/docset.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "Agora Objective-C",
-    "version": "2.9.0",
+    "version": "3.0.0",
     "archive": "Agora_Objective-C.tgz",
     "author": {
         "name": "Agora",
         "link": "https://www.agora.io"
     },
     "aliases": [],
     "specific_versions": [
+        {
+            "version": "2.9.0",
+            "archive": "versions/2.9.0/Agora_Objective-C.tgz"
```

Comment:
You need to actually add this archive. Renaming/moving the txt file does nothing.